app/templates/components/card-panel.hbs

```diff
@@ -0,0 +1,5 @@
+<div class="col-md-4">
```

Comment:
Ember will already wrap this in a div, so if you don't want an extra div, you can put the outermost HTML element into the js:

```
...
tagName: "div",
classNames: ["col-md-4"],
...
```